lib/hystrix-stats.js

```diff
@@ -11,12 +11,6 @@ const hystrixStream = new Transform({
   }
 });
 
-// TODO: This number is somewhat arbitrary. However, we need to allow
-// a potentially large number of listeners on this transform stream
-// because all circuits are connected to it. In an application with
-// a large number of circuits (or tests using circuits), if left to
-// the default MAX_LISTENERS, the user will see errors.
-hystrixStream.setMaxListeners(100);
```

Comment:
I agree this should go, but some implementers may start seeing a warning. The default is 10, so anyone who has 11-100 instances in their app would start to see the log.

Reply:
Good point. But I wonder if that's not so terrible. Typically, this might indicate a bug. For example, I have seen cases where people have created a new circuit every time the function needs to be called. It's a gross misunderstanding of how things should work, but leaving this value at 100 might prevent the user from ever knowing something is wrong.

Reply:
Maybe add a bit in the README for those who get the warning and go searching for what it means. Something that outlines they're probably not properly using a singleton pattern, or if they do have a valid use case they can call `shutdown()` or set a high max listeners manually.